website/content/docs/v0.10/Single Board Computers/rockpi_4.md

```diff
@@ -55,3 +55,37 @@ Retrieve the admin `kubeconfig` by running:
 ```bash
 talosctl kubeconfig
 ```
+
+## Boot Talos from an SSD Drive
+
+> Note: this is only tested on Rock PI 4c
+
+Rock PI 4 has an M2 slot which supports NVMe disks. It is possible to run Talos without any SD cards right from that SSD disk.
+
+Pre-installed SPI loader won't be able to chain Talos u-boot stored on the SSD drive because it's too outdated.
+Official docs for booting from the SSD also propose using outdated SPI flash u-boot.
+
+To make it work it is necessary to update u-boot to a more recent version.
+Armbian u-boot for Rock PI 4c proved to be working: [https://users.armbian.com/piter75/](https://users.armbian.com/piter75/).
+
+### Steps
+
+- flash any OS to the SD card (can be Armbian for example).
+- download Armbian u-boot and update SPI flash:
```

Comment:
```suggestion
## Boot Talos from an SSD Drive

> Note: this is only tested on Rock PI 4c

Rock PI 4 has an M2 slot which supports NVMe disks.
It is possible to run Talos without any SD cards right from that SSD disk.

The pre-installed SPI loader won't be able to chain Talos u-boot on the SSD drive because it's too outdated.
The official docs on booting from the SSD also propose using an outdated SPI to flash u-boot.

Instead, it is necessary to update u-boot to a more recent version for this process to work.
The Armbian u-boot build for Rock PI 4c has been proven to work: [https://users.armbian.com/piter75/](https://users.armbian.com/piter75/).

### Steps

- Flash any OS to the SD card (can be Armbian for example).
- Download Armbian u-boot and update SPI flash:
```